app/Services/CustomEmojiService.php

```diff
@@ -134,13 +134,13 @@ public static function all()
 					return $q->groupBy('shortcode');
 				})
 				->get()
-				->map(function($emojo) {
-					$url = url('storage/' . $emojo->media_path);
+				->map(function($emoji) {
+					$url = url('storage/' . $emoji->media_path);
```

Comment:
sometimes things like this are brands, but I don't _think_ this is one of them...

Reply:
emojo is the "fediverse1" lingo.